packages/next/test/unit/fixtures/01-normalize-routes/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "next": "canary",
+    "react": "experimental",
+    "react-dom": "experimental"
+  },
+  "scripts": {
+    "vercel-build": "next build"
```

Comment:
```suggestion
    "build": "next build"
```

Reply:
Happy to change it, but does it matter? I probably copied it from here: https://github.com/vercel/vercel/blob/623e43f865e927406200849aa3d7b6a9e9c2dcfc/packages/next/test/fixtures/11-export-clean-urls/package.json.

Reply:
`vercel-build` is only needed if you already have a `build` command defined and want to override it.

In fact, you can even override `buildCommand` in vercel.json now so the `vercel-build` script is even less useful.